roles/lib_openshift/library/oc_adm_ca_server_cert.py

```diff
@@ -958,13 +962,13 @@ def _process(self, template_name, create=False, params=None, template_data=None)
 
         return self.openshift_cmd(['create', '-f', fname])
 
-    def _get(self, resource, rname=None, selector=None):
+    def _get(self, resource, name=None, selector=None):
         '''return a resource by name '''
         cmd = ['get', resource]
-        if selector:
-            cmd.append('--selector=%s' % selector)
-        elif rname:
-            cmd.append(rname)
+        if selector is not None:
```

Comment:
Is this a "one or the other" thing as well? If so, should it raise if *both* are provided? Same question for other similar operations.